# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/quota/libwrap_fix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./rquota_svc.c.orig	2005-12-15 13:12:09.000000000 +0100
+++ ./rquota_svc.c	2005-12-15 13:12:11.000000000 +0100
@@ -57,6 +57,9 @@
  */
 struct authunix_parms *unix_cred;
 
+// libwrap.a expects the caller to provide this variables
+int allow_severity, deny_severity;
+
 #define FL_SETQUOTA 1	/* Enable setquota rpc */
 #define FL_NODAEMON 2	/* Disable daemon() call */
 #define FL_AUTOFS   4	/* Don't ignore autofs mountpoints */
